website_updater.py

```diff
@@ -147,7 +147,7 @@ def calc_mers(responses: List[dict]):
     elif num_countries > 9:
         mers += 1
 
-    return mers
+    return f"{mers:.1f}"
```

Comment:
I'm not sure if I'm happy to do the formatting here, rather than in the templates, but honestly whatever.

Reply:
true dat

Reply:
While we're at it, we can add return value type adnotation.